scss/_orange-navbar.scss

```diff
@@ -0,0 +1,62 @@
+// Contents
+//
+// Supra bar
+
+
+// Supra bar
+//
+// Brand exclusive thin navbar that can be used with orange navbar
+.supra {
+  display: none;
+  padding-top: 0;
+  padding-bottom: 0;
+  border-bottom: $navbar-orange-border-bottom solid $gray-700;
+
+  @include media-breakpoint-up(md) {
+    display: flex;
+    .container {
+      padding-right: $navbar-orange-padding-x;
+      padding-left: $navbar-orange-padding-x;
+    }
+  }
+
+  .navbar-nav .nav-link {
+    padding: #{map-get($spacers, 2) + $navbar-supra-padding-y} $navbar-supra-padding-x $navbar-orange-link-padding-bottom;
+    font-size: $font-size-sm;
+    line-height: 1;
```

Comment:
I think we need to use line-height for small font size here https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/2028fd76720bebe61e8dd9e778117bc2a23a2531/scss/_variables.scss#L685 and so maybe rework padding

Reply:
@Nurovek: something like that could help (to test deeply):

```diff
diff --git a/scss/_orange-navbar.scss b/scss/_orange-navbar.scss
index 573507c2db..d19036dff4 100644
--- a/scss/_orange-navbar.scss
+++ b/scss/_orange-navbar.scss
@@ -22,9 +22,9 @@
   }

   .navbar-nav .nav-link {
-    padding: #{map-get($spacers, 2) + $navbar-orange-supra-padding-y} $navbar-orange-supra-padding-x $navbar-orange-link-padding-bottom;
+    padding: $navbar-orange-link-padding-top $navbar-orange-supra-padding-x $navbar-orange-link-padding-bottom;
     font-size: $font-size-sm;
-    line-height: 1;
+    line-height: $line-height-sm;

     &.active::before {
       right: $navbar-orange-supra-padding-x;
diff --git a/scss/_variables.scss b/scss/_variables.scss
index d1a7239ab0..a6b46dfaf4 100644
--- a/scss/_variables.scss
+++ b/scss/_variables.scss
@@ -1251,13 +1251,13 @@ $navbar-active-transition:             bottom $navbar-transition-duration $navba
 // scss-docs-start orange-navbar-variables
 $navbar-orange-padding-x:              .125rem !default;
 $navbar-orange-border-bottom-width:    1px !default;
-$navbar-orange-link-padding-bottom:    .75rem !default;
+$navbar-orange-link-padding-top:       .75rem !default;
+$navbar-orange-link-padding-bottom:    .6875rem !default;
 $navbar-orange-icon-padding-top:       divide($spacer, 2) !default;
 $navbar-orange-icon-padding-x:         .9375rem !default;
 $navbar-orange-icon-padding-bottom:    .25rem !default;
 $navbar-orange-badge-padding-y:        .125rem !default;
 $navbar-orange-badge-padding-x:        .375rem !default;
-$navbar-orange-supra-padding-y:        map-get($spacers, 4) * .1 !default; // 0.1875rem
 $navbar-orange-supra-padding-x:        map-get($spacers, 1) * 1.5 !default;
 $navbar-orange-supra-margin-y:         map-get($spacers, 4) * .1 !default; // 0.1875rem
 $navbar-orange-supra-icon-size:        1.5625rem !default;
```